pinot-core/src/main/java/org/apache/pinot/core/segment/index/loader/defaultcolumn/BaseDefaultColumnHandler.java

```diff
@@ -303,6 +304,10 @@ protected void createColumnV1Indices(String column)
         dictionaryElementSize = StringUtil.encodeUtf8(stringDefaultValue).length;
```

Comment:
we should use something that is independent of the encoding on the system. The file might be created/saved in a different encoding right

Reply:
@kishoreg We always encode String using UTF-8. As long as the files are generated with Pinot library, the String will be encoded with UTF-8. Record reader should be able to decode other format if necessary.